src/Hng.Domain/Hng.Domain.csproj

```diff
@@ -22,7 +22,12 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.7">
```

Comment:
Please remove all Entityframework dependencies from domain

Reply:
I removed other Entity framework dependencies, except Entity Framework Core because the absence of it is making my build fail.